### PR TITLE
Build new ubuntu advantage service description

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build new uasd at `/legal/ubuntu-advantage-service-description`
- Build new table of contents

## QA

- View page at : https://ubuntu-com-10402.demos.haus/legal/ubuntu-advantage-service-description
- Check against [copy doc](https://docs.google.com/document/d/15URkXf2vh7bv89n-cWQPeaP2EehSNUnUCoZvGbNTYFA/edit#heading=h.eddnsno5g76l)
- Check the table of contents works as expected

## Issue / Card

Fixes [#4435](https://github.com/canonical-web-and-design/web-squad/issues/4435)

### CODE REVIEW

File: templates/legal/ubuntu-advantage-service-description/_toc.html [hunk 1]
```diff
@@ -2,27 +2,80 @@
...
+        </ul>
+      </li>
+      <li><a href="#uasd-support-services-process">UA Support Services Process</a></li>
       <li><a href="#uasd-definitions">Definitions</a></li>
     </ul>
   </nav>
 </div>
 
-<div class="p-table-of-contents__section">
+<!-- Removed for now - can replase once japanese translation is done-->
```
Comment (petermakowski) [inline templates/legal/ubuntu-advantage-service-description/_toc.html:73]:
```suggestion
<!-- Removed for now - can replace once japanese translation is done-->
```

**Review by petermakowski — APPROVED** [review]
LGTM

File: templates/legal/ubuntu-advantage-service-description/_toc.html (hunk 1 above)
Comment (bethcollins92) [reply templates/legal/ubuntu-advantage-service-description/_toc.html:73]: 😅 thanks

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 2]
```diff
@@ -4,386 +4,390 @@ context:
...
+<h2 id="uasd-definitions">Definitions</h2>
 
-1. Desktop support services are subject to the exceptions listed below.
-2. Scope. Desktop support covers all packages in the base Ubuntu desktop image and packages necessary for basic network authentication and connectivity using sssd, winbind, network-manager, and network-manager plugins in the Ubuntu Main repository. It also covers Ubuntu distribution for Windows Subsystem for Linux (WSL).
-3. Out of scope.
-4. Dual-booting (cohabitating with other operating systems).
-5. Peripherals which are not certified to work with Ubuntu.
-6. Community flavours of Ubuntu.
-7. Support for architectures other than x86_64.
+**Applications:** the supported Applications as set out at [https://ubuntu.com/support](https://ubuntu.com/support) [which are built by Canonical or as otherwise agreed with Canonical. Or Managed Applications as described in the Add-ons section, under Managed Services, and at https://ubuntu.com/managed](https://ubuntu.com/managed).[ ](https://ubuntu.com/managed) The list of Applications are subject to change without notice
```
Comment (carkod) [inline templates/legal/ubuntu-advantage-service-description/index.md:331]: the link is not this entire thing, remove the brackets. Also `[ ](https://ubuntu.com/managed)` doesn't look like correct markdown

File: templates/shared/_ubuntu-advantage-service-description.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            <td class="u-align--center">1h</td>
+          </tr>
+          <tr>
+            <td><a href="/legal/ubuntu-advantage-assurance">Ubuntu assurance program</a></td>
+            <td class="u-align--center">No</td>
+            <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg", alt="Yes", width="16", height="16", hi_def=True, loading="lazy",) | safe }}</td>
+            <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg", alt="Yes", width="16", height="16", hi_def=True, loading="lazy",) | safe }}</td>
+          </tr>
+          <tr>
+            <td><a href="/esm">ESM</a></td>
```
Comment (petermakowski) [inline templates/shared/_ubuntu-advantage-service-description.html:32]: Shouldn't this be "Extended Security Maintenance (ESM)"?

File: templates/legal/ubuntu-advantage-service-description/index.md (hunk 2 above)
Comment (bethcollins92) [reply templates/legal/ubuntu-advantage-service-description/index.md]: Good spot thanks @carkod - fixed

File: templates/shared/_ubuntu-advantage-service-description.html (hunk 3 above)
Comment (bethcollins92) [reply templates/shared/_ubuntu-advantage-service-description.html]: it should! thanks

**Comment by bethcollins92** [comment]
Added the +1 labels myself as @SirSamTumless and @lechsandecki have QA'd and Ok'd it

**Pull request merged**